testbed/Makefile

```diff
@@ -0,0 +1,51 @@
+GOTEST_OPT?=-v -race -timeout 30s
```

Comment:
Some of the options and target need to be custom but perhaps it is worth to check the ones that can be the ones already defined in Makefile.Common

Reply:
Done.